charts/opentelemetry-collector/templates/prometheusrule.yaml

```diff
@@ -10,5 +10,78 @@ metadata:
     {{- end }}
 spec:
   groups:
+{{- if .Values.prometheusRule.groups }}
     {{- toYaml .Values.prometheusRule.groups | nindent 4 }}
 {{- end }}
+{{- if .Values.prometheusRule.defaultRules.enabled }}
+  - name: collectorRules
+    rules:
+    - alert: ReceiverDroppedSpans
+      expr: rate(otelcol_receiver_refused_spans[5m]) > 0
+      for: 2m
+      labels:
+        severity: critical
+      annotations:
+        description: '{{`The {{ $labels.receiver }} is dropping spans at a rate of {{ humanize $value }} per second `}}'
+        runbook_url: 'https://github.com/open-telemetry/opentelemetry-collector/blob/main/docs/monitoring.md#receive-failures'
+    - alert: ReceiverDroppedMetrics
+      expr: rate(otelcol_receiver_refused_metric_points[5m]) > 0
+      for: 2m
+      labels:
+        severity: critical
+      annotations:
+        description: '{{`The {{ $labels.receiver }} is dropping metrics at a rate of {{ humanize $value }} per second `}}'
+        runbook_url: 'https://github.com/open-telemetry/opentelemetry-collector/blob/main/docs/monitoring.md#receive-failures'
+    - alert: ProcessorDroppedSpans
+      expr: rate(otelcol_processor_dropped_spans[5m]) > 0
+      for: 2m
+      labels:
+        severity: critical
+      annotations:
+        description: '{{`The {{ $labels.processor }} is dropping spans at a rate of {{ humanize $value }} per second `}}'
+        runbook_url: 'https://github.com/open-telemetry/opentelemetry-collector/blob/main/docs/monitoring.md#data-loss'
+    - alert: ProcessorDroppedMetrics
+      expr: rate(otelcol_processor_dropped_metric_points[5m]) > 0
+      for: 2m
+      labels:
+        severity: critical
+      annotations:
+        description: '{{`The {{ $labels.processor }} is dropping metrics at a rate of {{ humanize $value }} per second `}}'
+        runbook_url: 'https://github.com/open-telemetry/opentelemetry-collector/blob/main/docs/monitoring.md#data-loss'
+    - alert: ExporterDroppedSpans #metric not present in collector 0.54
```

Comment:
If this isn't present in 0.54.0 by design then I don't think we should include it as a default rule since the chart's default collector version is 0.54.0

Reply:
Problem is that it's present in the docs ... I need to do a bit of digging

Reply:
The monitoring.md doc hasn't been updated in 5 months so I don't really trust it lol

Reply:
deleted alert